plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/constants/CodeTransformFileConstants.kt

```diff
@@ -0,0 +1,24 @@
+// Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.codemodernizer.constants
+
+const val HIL_POM_VERSION_PLACEHOLDER = "*****"
+
+const val HIL_ARTIFACT_DIR_NAME = "q-hil-dependency-artifacts"
```

Comment:
Note: Verify this directory is cleaned up at end of HIL. Close this out.

Reply:
Yes, the `hilCleanup` function removes the temp directory.